call_server/static/scripts/site/search.js

```diff
@@ -74,15 +74,15 @@
         });
 
       } else {
-        // hit Sunlight OpenStates
+        // hit OpenStates
 
         // TODO, request state metadata
         // display latest_json_date to user
 
         $.ajax({
-          url: CallPower.Config.SUNLIGHT_STATES_URL,
+          url: CallPower.Config.OPENSTATES_URL,
           data: {
-            apikey: CallPower.Config.SUNLIGHT_API_KEY,
+            apikey: CallPower.Config.OPENSTATES_API_KEY,
             state: campaign_state,
```

Comment:
To work with the [v3 api](https://v3.openstates.org/docs#/people/people_search_people_get) it looks like some of these params need to change

`state` -- became `jurisdiction`

`last_name` -- became just `name`

`in_office` -- seems to not be supported at all anymore

`chamber` -- became `org_classification` and should be one of "legislature", "executive", "lower", "upper", or "government"

Reply:
Down below in the `renderSearchResults` function, some translation is also needed to hook the v3 api response up to the template in [call_server/templates/campaign/form.html:167](https://github.com/EFForg/call-congress/blob/c9400b075131dc3be3b7b57cfc349bf4a56f2d6a/call_server/templates/campaign/form.html#L167), which still expects the v1 attributes. This worked for me:

```diff
         } else if (person.title === 'Governor') {
           person.uid = 'us_state:governor:'+person.state
         }
 
+        if (person.given_name) {
+          person.first_name = person.given_name;
+          person.last_name = person.family_name;
+          person.title = person.current_role.title;
+          person.state = person.jurisdiction.name;
+        }
+
         // if person has multiple phones, use only the first office
         if (person.phone === undefined && person.offices) {
           if (person.offices) {
-            person.phone = person.offices[0].phone;
+            person.phone = person.offices[0].voice;
           }
```